run.sh

```diff
@@ -98,6 +103,11 @@ if [ -z "$GALAXY_CONFIG_FILE" ]; then
     export GALAXY_CONFIG_FILE
 fi
 
+if [ $INITIALIZE_TOOL_DEPENDENCIES -eq 1 ]; then
+    # Install Conda environment if needed.
+    python ./scripts/manage_tool_dependencies.py -c "$GALAXY_CONFIG_FILE" init_if_needed
```

Comment:
I really don't think so - this is of no use to the reports app or the tool shed for instance.